webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyStatePrimaryActionButton.js

```diff
@@ -23,14 +23,19 @@ const PrimaryActionButton = ({ action }) => {
 
 // eslint-disable-next-line react/prop-types
 const urlButton = ({ url, title }) => (
-  <Button href={url} bsStyle="primary" bsSize="large">
+  <Button component="a" href={url} target="_blank" variant="primary">
```

Comment:
Why using `target="_blank"`? doesn't this link lead to another page in foreman?

Reply:
I think that is responsible to open the link in a new tab.

Reply:
indeed, but is there a reason for opening it in a different tab?  doesn't it a link to another page in foreman?